### DISCUSSION TITLE
How do I exit review properly?

### DISCUSSION THREAD

**Discussion by alg** [harrisoncramer/gitlab.nvim#105] [Q&A] [answered]
Hi there,

After I started the review session, how do I exit it properly? I've seen it in demo video that you closed it quickly, but I don't see the shortcuts for this. At the moment I'm closing three panes with ":q", but guess it's not how it's supposed to be.

**Comment by harrisoncramer** [comment, marked as answer]
The reviewer is just a tab so you can close it with `:tabclose`

I've got a keybinding for that on my machine (`<leader>tc`) which is why you don't see it in the video.
